node/internal/yarn_repository.bzl

```diff
@@ -28,8 +28,17 @@ def _yarn_repository_impl(ctx):
     modules_path = ctx.path("node_modules")
     execute(ctx, ["find", modules_path, "-iname", "build", "-type", "f", "-exec", "mv", "{}", "{}.js", ";"])
     modules = []
+    # Rename and move scoped modules with invalid bazel labels
+    for module in modules_path.readdir():
+      if module.basename.startswith("@"):
+        for scoped_module in module.readdir():
+          execute(ctx, ["mv", scoped_module, "%s/%s" % (modules_path, mangle_package_name(module.basename, scoped_module.basename))])
+
     for module in modules_path.readdir():
         if module.basename.startswith("."): continue
+        # scoped modules just contain other modules so don't include them
```

Comment:
This check appears to be unnecessary as the rename will take care of it?

Reply:
The scoped modules are moved out of the parent directory but the parent - with the `@` prefix` - remains and needs to be skipped. (It's empty)